x/gov/types/config.go

```diff
@@ -4,11 +4,14 @@ package types
 type Config struct {
 	// MaxMetadataLen defines the maximum proposal metadata length.
 	MaxMetadataLen uint64
+	// MaxOptionsLen defines the maximum number of options a proposal can have.
+	MaxOptionsLen uint64
```

Comment:
Would you be up for opening an issue on Cosmos Hub asking for this?
This seems to me can be implemented in vanilla SDK as more people may want to use it.
If they agree to upstream it, you can get rid of it in your fork in a future version.

Reply:
Right, this could help us indirectly. Done [here](https://github.com/cosmos/cosmos-sdk/issues/20074). Had to change it slightly as - in the meantime - configs for `gov` changed quite a bit.

Reply:
Thanks a lot @marcello33, I took a look at the issue when you shared it via slack, and it's looking great.
Resolved